src/components/profile/ViewOrUploadAvatar/index.js

```diff
@@ -39,32 +40,23 @@ const ViewOrUploadAvatar = props => {
   const wrappedUserStorage = useWrappedUserStorage()
   const [showErrorDialog] = useErrorDialog()
 
-  const handleCameraPress = useCallback(
-    event => {
-      event.preventDefault()
+  const handleCameraPress = useOnPress(() => {
+    openCropper({
+      pickerOptions,
+      navigation: props.navigation,
+      wrappedUserStorage,
+      showErrorDialog,
+      log,
+      avatar: profile.avatar,
+    })
+  }, [navigation, wrappedUserStorage, showErrorDialog, profile])
```

Comment:
1. destructurize profile above hook e.g. const { avatar } = profile
2. pass just 'avatar' on line 50
3. use jusr 'avatar' in the deps